plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/PinotQueryBuilder.java

```diff
@@ -81,7 +84,11 @@ public static String generatePql(PinotTableHandle tableHandle, List<PinotColumnH
         }
         pqlBuilder.append(" LIMIT ")
                 .append(limit);
-        return pqlBuilder.toString();
+        String pql = pqlBuilder.toString();
+        if (queryLoggingEnabled && log.isInfoEnabled()) {
+            log.info("Generated PQL: %s", pql);
```

Comment:
Change to log.debug.

Reply:
Debug logs are not enabled. If we want to see logs with single config its better to make it info. By default its false and you can enable / disable it per catalog.

Reply:
Disagreed. We don't manage log settings in catalog files.

Reply:
This can lead to a log flood. Please change to debug level.

Reply:
I have made these debug. But the logs are not enabled by default. It has to be configured to be enabled. Also the current approach surely leads to bigger flood of logs as enabling the logger enables for all catalogs and there is no per catalog setting. On top of this enables debug for the entire class not just one log statement. But since you are the moderator I yield.